NitroxServer/Communication/Packets/Processors/BulkheadDoorStateChangedProcessor.cs

```diff
@@ -0,0 +1,18 @@
+using NitroxServer;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.GameLogic;
+
+public class BulkheadDoorStateChangedProcessor : AuthenticatedPacketProcessor<BulkheadDoorStateChanged>
```

Comment:
As explained in your other pr if no processor is there the systemd defaults to this behaviour. Therefor this can be safely deleted.